docs/zh_cn/04-developer-guide/do_regression_test.md

```diff
@@ -210,9 +233,10 @@ models:
 - [x] TorchScript
 - [x] MMDeploy SDK
 
-## 5. 支持的Codebase及其Metric
+## 6. 支持的Codebase及其Metric
```

Comment:
How about making a metric table: {codebase} x {all kinds of metric}?

Reply:
Done .

| Codebase | Metric | Support           |
|----------| ---------- |-------------------|
| mmdet    | bbox | :heavy_check_mark: |
|     | segm | :heavy_check_mark: |
|     | PQ | :x:                |
|  mmcls   | accuracy | :heavy_check_mark: |
|  mmseg   | mIoU | :heavy_check_mark: |
|  mmpose   | AR | :heavy_check_mark: |
|     | AP | :heavy_check_mark: |
|   mmocr  | hmean | :heavy_check_mark: |
|     | acc | :heavy_check_mark: |
|   mmedit  | PSNR | :heavy_check_mark: |
|     | SSIM | :heavy_check_mark: |

Reply:
It got stuck when I execute the following command:
```shell
 python tools/regression_test.py --codebase mmdet --backends tensorrt sdk --work-dir ~/Data/test --performance
```
Part of the log is shown as below,
```shell
Traceback (most recent call last):
  File "./tools/deploy.py", line 306, in <module>
    main()
  File "./tools/deploy.py", line 193, in main
    ret_value=ret_value)
  File "./tools/deploy.py", line 60, in create_process
    process.join()
  File "/home/PJLAB/lvhan/miniconda3/envs/mmdeploy/lib/python3.7/multiprocessing/process.py", line 140, in join
    res = self._popen.wait(timeout)
  File "/home/PJLAB/lvhan/miniconda3/envs/mmdeploy/lib/python3.7/multiprocessing/popen_fork.py", line 48, in wait
    return self.poll(os.WNOHANG if timeout == 0.0 else 0)
  File "/home/PJLAB/lvhan/miniconda3/envs/mmdeploy/lib/python3.7/multiprocessing/popen_fork.py", line 28, in poll
    pid, sts = os.waitpid(self.pid, flag)

```

Reply:
It seems to be a zombie thread. Were there any script processes at the same time besides regression test ? Or were you stop the thread with `ctrl+c` when process some backend convertion? In my case I can't kill the thread when converting pplnn, I need to kill it with pid.

Reply:
No other script is processing. 
I didn't use ctrl+c either.

Reply:
Is it always stuck or just one time? 
This is the first time I met this issue.

Reply:
> Is it always stuck or just one time? This is the first time I met this issue.

Seems always

Reply:
Can you describe the specific steps so that I can repeat this issue?

Reply:
![image](https://user-images.githubusercontent.com/4560679/169734386-f1ff60d9-a2ba-4dc1-8e1a-f02c4feb563a.png)

Reply:
Pls check if you can do it with cmd, you can find some like `Process cmd: xxx`, pls use terminal to process it see if it can process sucessfully. I think we need to confirmif that first.

Reply:
I found it got stuck sometimes when only executing the converting command. Odd. 
I am gonna to check my environment.

Reply:
Yes. Executing the conversion command is successful. 
```shell
python3 ./tools/deploy.py /home/PJLAB/lvhan/Documents/projects/open-mmlab/mmdeploy/configs/mmdet/detection/detection_tensorrt-fp16_dynamic-320x320-1344x1344.py /home/PJLAB/lvhan/Documents/projects/open-mmlab/mmdetection/configs/retinanet/retinanet_r50_fpn_1x_coco.py "/home/PJLAB/lvhan/Documents/projects/open-mmlab/mmdeploy_checkpoints/mmdet/RetinaNet/retinanet_r50_fpn_1x_coco_20200130-c2398f9e.pth" "../mmdetection/demo/demo.jpg" --work-dir "/home/PJLAB/lvhan/Data/test/mmdet/RetinaNet/tensorrt/dynamic/retinanet_r50_fpn_1x_coco_20200130-c2398f9e" --device cuda --log-level INFO --dump-info --test-img ./tests/data/tiger.jpeg
```
That cost about 6 minutes.